manifests/p/ProaData/B2BClient/1.1.0/ProaData.B2BClient.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: ProaData.B2BClient
+PackageVersion: 1.1.0

```

Comment:
```suggestion
PackageVersion: 1.1.0.0
```

Reply:
Una centralita All in One, con la API oficial de WhatsApp. Su propia PBX de mensajería. Permite gestionar toda la interacción del negocio con sus clientes en un solo lugar. Todas las conversaciones con sus clientes integradas en su CRM, ERP, DMS...

Una aplicación multicanal para el control de sus comunicaciones:
* Todas las conversación en un solo lugar
* API oficial WhatsApp
* Administración Web
* Integración con API de CRM
* Pago por uso